config/initializers/puma_worker_killer.rb

```diff
@@ -0,0 +1,3 @@
+PumaWorkerKiller.config do |config|
```

Comment:
Do we need a `before_fork` hook, or a call to `start`? Would the gem be activated this way?

Reply:
Yeah I'm pretty sure Bilka is right. Based on the docs (which I am having a little trouble following since there seem to be multiple paths to configuring it), we need to do this in Ansible instead. The linked PR (visible only to folks with read+ access to that repo) needs to do it instead, since we don't have `config/puma.rb` here

Reply:
I was expecting to do it in the config/puma.rb